src/settings.ts

```diff
@@ -48,6 +48,13 @@ export enum METRIC_NAMES {
   ANCHOR_REQUESTS_BATCH_TIME = 'anchor_requests_batch_time',
   ANCHOR_REQUESTS_BATCH_FAILURE_TIME = 'anchor_requests_batch_failure_time',
 
+
+  // *******************************************************************//
+  // Ethereum Blockchain Service
+
+  ETH_REQUEST_TOTAL = 'eth_request_total',
```

Comment:
What does request stand for? Based on the code it seems that it would be equal to the number of anchors we attempt. Is that what we want?

Reply:
@stephhuynh18 so the request was to track the number of requests to the ethereum endpoint (and added on errors also).  That probably does match the number of anchor batches we attempt, tho not sure if it matches the total of individual anchors - don't we batch them into eth requests?  

The concern here was to track eth requests and failures, also velocity, i think this came up when we ran out of funds in the wallet

Since the concern is with the eth wallet maybe we shoudl track here, so if we change the batching logic, we still will track specifically the eth requests?

Reply:
@stbrody this was originally your request i think, is that a good summary of why? ^

Reply:
per anchor we hit the ethereum endpoint many times (get balance, get nonce, send raw transaction). Are we trying to track each individual call. Also are counting the number of requests where it is now, does not take into account the retries

Reply:
i'm not sure what the exact purpose was actually.  @stbrody do you know what the exact desired metric is?  each time we hit the endpoint? do we want to count the retries?  I'll ask this in the ticket as well

Reply:
aww geez, this request was from over 3 months ago now so I don't really remember the context.  Looking at the ticket, the motivating factor seemed to be we were hitting infura rate limits.  I mean, I guess in an ideal world we'd have a separate metric to track each type of call we make to the eth rpc endpoint (chainid, get balance, get nonce, send transaction, etc), then we could sum up all those separate metrics to see the overall total number of requests of any type.

Reply:
ok, i think maybe one metric with multiple labels (a label for each type of call) would be right then